### ISSUE TITLE
Layer surgery

### ISSUE THREAD

**Issue by cpadavis** [cpadavis/billy#26]
For transfer learning, we'll need to modify the final layer from galaxy zoo outputs to space warps.

**Comment by ajmcleod** [comment]
This is now possible using transfer_learn_conv_net() instead of load_conv_net(). The only extra argument you need to provide compared to load_conv_net() is cost_function = 'log_likelihood' or 'rms_error', because the cost function will change as we move from galaxy zoo data to space warps data

**Comment by ajmcleod** [comment]
I should mention that transfer_learn_conv_net() makes the minimum change possible for transfer learning---training will now be on the new set of images/labels you've provided, but it will train all layers as before. You can turn off training in the early layers by setting 'learning_rate' to 0.0 in the part of network_architecture corresponding to that layer

**Comment by drphilmarshall** [comment]
Nice! Are you thinking of training different numbers of layers and
comparing results, or do only certain choices make sense?

**Comment by ajmcleod** [comment]
The normal choice would be to turn off training on all of the convolution layers, but if we have time I think it's worth seeing if performance improves when we retrain these layers as well (possibly at a reduced learning rate).
